scripts/Dockerfile.base

```diff
@@ -22,6 +22,25 @@ RUN apk add --no-cache \
 		-G coder \
 		coder
 
+# Install Terraform plugins
+RUN mkdir -p /opt/terraform/plugins/registry.terraform.io
+# Add config for local terraform
+ADD files/terraform-config.tfrc /opt/terraform/config.tfrc
+ARG CODER_PROVIDER_VERSION=0.6.12
```

Comment:
These versions (for Coder and the provider) can also get quickly out of sync from what we hardcode in our example templates too, which will cause Terraform to still fail

Maybe we stick to just embedding Terraform and make the TRFC configurable via a coder-server flag or something? Then, an external PVC can be mounted or a network mirror can be specified https://github.com/coder/coder/issues/6189

Reply:
Yeah, that sounds like a better option. Baking in all of the Terraform providers we use in `examples/templates` would inflate our base image by 199MB.